### PR TITLE
Display hero in html

### PR DESCRIPTION
**Pull request by jhcastanod**

### CODE REVIEW

File: src/app/services/hero/hero.service.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 @Injectable({
   providedIn: 'root'
 })
 export class HeroService {
-  readonly endpoint = 'http://www.mocky.io/v2/5e84fa50300000460a97ad84';
+  readonly endpoint = 'http://www.mocky.io/v2/5e86445c3100000929813940';
 
-  constructor(private readonly http: HttpClient) {}
+  constructor(private readonly http: HttpClient) { }
```
Comment (luqezman) [inline src/app/services/hero/hero.service.ts:16]: ` constructor(private readonly http: HttpClient) {}`

File: src/app/app.component.ts [hunk 2]
```diff
@@ -20,7 +21,8 @@ export class AppComponent implements OnInit {
       error: (error) => {
         console.log('heroService.error', error);
       },
-      next: () => {
+      next: (hero) => {
+        this.hero = hero;
         console.log('next');
```
Comment (luqezman) [inline src/app/app.component.ts:26]: switch to `console.log(this.hero);`

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,21 +1,8 @@
-<!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
+  <h1>Name: {{hero?.name}}</h1>
+  <h3>Life: {{hero?.life}}</h3>
+  <p>Fly: {{hero?.fly}}</p>
   <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
```
Comment (luqezman) [inline src/app/app.component.html:5]: Remove that image

File: src/app/app.component.ts [hunk 4]
```diff
@@ -20,7 +21,8 @@ export class AppComponent implements OnInit {
       error: (error) => {
         console.log('heroService.error', error);
       },
-      next: () => {
+      next: (hero) => {
```
Comment (luqezman) [inline src/app/app.component.ts:24]: Change to `next: (hero: IHero) => {`

Comment (jhcastanod) [reply src/app/app.component.ts:24]: why this change if the server can send any other type?

Comment (luqezman) [reply src/app/app.component.ts:24]: Pipes are called before responses

File: src/app/interceptors/bad-request.interceptor.ts [hunk 5]
```diff
@@ -11,7 +11,7 @@ import { catchError } from 'rxjs/operators';
 
 import { BadResquestErrorsToFormErrors } from 'transformers/bad-request-errors-to-form-errors.transformer';
 
-export class BadRequestHttpInterceptor implements HttpInterceptor {
+export class BadRequestInterceptor implements HttpInterceptor {
   // TODO: Fix type
```
Comment (luqezman) [inline src/app/interceptors/bad-request.interceptor.ts:15]: change to `// TODO: Fix return type`

File: src/app/app.component.ts [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
 import { HeroService } from 'services/hero/hero.service';
+import { IHero } from 'interfaces/hero';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
+  hero: any;
```
Comment (luqezman) [inline src/app/app.component.ts:12]: hero: IHero;

File: src/app/app.component.ts [hunk 7]
```diff
@@ -1,13 +1,15 @@
 import { Component, OnInit } from '@angular/core';
 
 import { HeroService } from 'services/hero/hero.service';
```
Comment (luqezman) [inline src/app/app.component.ts:3]: Sort imports

File: src/app/app.component.ts (hunk 6 above)
Comment (luqezman) [inline src/app/app.component.ts:12]: `hero: IHero;`

File: src/app/interceptors/internal-server-error.interceptor.ts [hunk 8]
```diff
@@ -5,10 +5,10 @@ import { catchError } from 'rxjs/operators';
 
 import { NotificationService } from 'services/notification/notification.service';
 
-export class InternalServerErrorHttpInterceptor implements HttpInterceptor {
+export class InternalServerErrorInterceptor implements HttpInterceptor {
   constructor(private readonly notificationService: NotificationService) {}
 
-  // TODO: fix type
+  // TODO: Fix type
```
Comment (luqezman) [inline src/app/interceptors/internal-server-error.interceptor.ts:11]: change to  `// TODO: Fix parameter/return type`

File: src/app/app.component.ts [hunk 9]
```diff
@@ -20,8 +22,9 @@ export class AppComponent implements OnInit {
       error: (error) => {
```
Comment (luqezman) [inline src/app/app.component.ts:22]: Remove error callback

**Pull request merged**